spec/system/provider_interface/provider_can_view_status_of_active_applications_report_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+RSpec.feature 'View active status of applications report' do
+  include CourseOptionHelpers
+  include DfESignInHelpers
+
+  let(:provider) { create(:provider, :with_signed_agreement) }
+  let(:other_provider) { create(:provider, :with_signed_agreement) }
+  let(:course_without_accredited_provider) { create(:course, name: 'Beekeeping', provider: provider, accredited_provider: nil) }
+  let(:course_option_without_accredited_provider) { create(:course_option, course: course_without_accredited_provider) }
+  let(:course_with_other_accredited_provider) { create(:course, name: 'Archaeology', provider: provider, accredited_provider: other_provider) }
+  let(:course_option_with_other_accredited_provider) { create(:course_option, course: course_with_other_accredited_provider) }
+  let(:course_provider_accredits) { create(:course, name: 'Criminology', provider: other_provider, accredited_provider: provider) }
+  let(:course_option_provider_accredits) { create(:course_option, course: course_provider_accredits) }
+
+  before do
+    FeatureFlag.activate(:provider_reports_dashboard)
+  end
+
+  scenario 'a provider can navigate, view and export the active applications status report' do
+    given_i_am_a_provider_user_with_dfe_sign_in
+    and_i_am_permitted_to_see_applications_for_my_provider
+    and_my_organisation_has_courses_with_applications
+    and_i_sign_in_to_the_provider_interface
+    when_i_visit_the_reports_page
+    and_i_click_on_the_status_of_active_applications_report
+    then_i_can_view_the_active_course_data_for_my_provider
+    and_i_can_download_the_data_as_a_csv
+  end
+
+  def given_i_am_a_provider_user_with_dfe_sign_in
+    provider_exists_in_dfe_sign_in
+  end
+
+  def and_i_am_permitted_to_see_applications_for_my_provider
+    @provider_user = provider_user_exists_in_apply_database
+    @provider_user.update(providers: [provider])
+  end
+
+  def and_my_organisation_has_courses_with_applications
+    create_list(:application_choice, 10, status: :interviewing, course_option: course_option_with_other_accredited_provider)
```

Comment:
We're creating multiple records here but we don't seem to be expecting/asserting anything about multiples eg. counts.
Is the record creation necessary in this volume?

Reply:
It isnt necessary,  the setup was copied from an earlier spec and can be simplified.